app/controllers/api/v1/votes_controller.rb

```diff
@@ -22,6 +22,14 @@ def destroy
     end
   end
 
+  def index
```

Comment:
that's not true many of our api/v1 endpoints are /v1/collection?by_some_scope_id=123